corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -21,6 +21,8 @@
     var show_inactive = {{ show_inactive|yesno:"true,false" }};
     var alert_user = hqImport("hqwebapp/js/alert_user").alert_user;
 
+    var location_toggle = {{ request|toggle_enabled:'LOCATION_SEARCH'|yesno:"true,false"}};
```

Comment:
Check out https://github.com/dimagi/js-guide/blob/master/integration-patterns.md#toggles-and-feature-previews? We made that pattern exactly for this situation :)